packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,479 @@ Before installing Chainweb, you should verify that your computer meets the follo
 - Network: Publicly-accessible IP address.
 - Operating system: Linux AMD64 architecture.
 
-If you also plan to use the node as an API server for Pact or for mining, Rosetta API integration, or accessing indexed data using `chainweb-data` you should verify that your computer meets the following minimum recommended hardware requirements: 
+If you also plan to use the node as an API server for Pact calls or for mining, exchange integration, or accessing indexed data, you should verify that your computer meets the following minimum recommended hardware requirements: 
 
 - Four (4) CPU cores.
 - Eight (8) GB of RAM.
 
+### Operating system architecture
+
+Most `chainweb-node` software runs on Linux or macOS with AMD64 architectures.
+If you want to run a Chainweb node on Linux or macOS with ARM64 architecture, you'll need to build the binaries yourself from the source code.
+You might experience some unexpected behavior when running a node on ARM64 architecture.
+If you want to run a Chainweb node on Microsoft Windows, you should note that this isn't a supported configuration and requires you to install the latest Windows Services for Linux (WSL) software.
+If you choose to proceed after installing WSL, you can install the `chainweb-node` software in the WSL virtual environment rather than using the native Windows operating system. 
+
+### Incoming and outgoing network rules
+
+In addition to a publicly-accessible IP address, nodes must be able to communicate with peers and accept incoming messages from other computers in the network.
+If your computer is behind a firewall or is a virtual guest connecting to the internet through a host computer, you should review your network configuration and open required ports to allow inbound and outbound traffic. 
+For example, you might need to configure port forwarding for port 1789 on your router or host computer to allow inbound connections from remote nodes.
+
 ## Installation options
 
 There are several options for setting up a Chainweb node in a physical or virtual environment.
 For example, you can run a Chainweb node image in a Docker container or build the binaries directly from the source code.
+You can install release binaries directly on a physical Linux server or run them using the infrastructure from a cloud services provider.
+
+For more information about your installation options, see the following topics:
+
+- [Install release binaries](#install-release-binaries)
+- [Run in a Docker container](#run-in-a-docker-container)
+- [Build from source code](#build-from-source)
+
+## Install release binaries
+
+You can download compressed archive files with `chainweb-node` release binaries for Ubuntu Linux directly from the [Releases](https://github.com/kadena-io/chainweb-node/releases) pages in the [chainweb-node](https://github.com/kadena-io/chainweb-node/) repository.
```

Comment:
```suggestion
You can download compressed archive files with `chainweb-node` release binaries for Ubuntu Linux directly from the [Releases](https://github.com/kadena-io/chainweb-node/releases) page in the [chainweb-node](https://github.com/kadena-io/chainweb-node/) repository.
```
We mark old releases as private after we make a new release, because ordinarily there is no reason to use an older version. Idk if that deserves a note, but it does mean there's only one release page at any time.